core/nodewallets/commander.go

```diff
@@ -210,6 +210,14 @@ func wrapPayloadIntoInputData(data *commandspb.InputData, cmd txn.Command, paylo
 		} else {
 			panic("failed to wrap StateVariableProposal")
 		}
+	case txn.RotateEthereumKeySubmissionCommand:
+		if underlyingCmd, ok := payload.(*commandspb.EthereumKeyRotateSubmission); ok {
+			data.Command = &commandspb.InputData_EthereumKeyRotateSubmission{
+				EthereumKeyRotateSubmission: underlyingCmd,
+			}
+		} else {
+			panic("failed to wrap RotateEthereumKeySubmissionCommand")
+		}
```

Comment:
Is that needed? we do not send the transaction from a running node are we?

Reply:
oh but it's used from the command line where we instantiate a commander most likely..

Reply:
Haha yea I'm not sure what your question is :p but what I do know is that the commander panics here if we don't do this.